src/tokenize_cleanup.cpp

```diff
@@ -1068,3 +1055,73 @@ static void check_template(chunk_t *start)
            (end != NULL) ? get_token_name(end->type) : "<null>");
    set_chunk_type(start, CT_COMPARE);
 } // check_template
+
+
+void cleanup_objc_property_with_parens(chunk_t *start)
+{
+   chunk_t *next = chunk_get_next_ncnl(start);
+
+   set_chunk_parent(next, start->type);
```

Comment:
I am not entirely sure, but this looks like it could need a nullptr check for next.